website/docs/components/input-tags/examples/basic.jsx

```diff
@@ -50,14 +56,20 @@ const Demo = () => {
             editable
             data-id={idx}
             onClick={handleEditTag}
+            onKeyDown={handleKey}
           >
             <InputTags.Tag.Text>{tag}</InputTags.Tag.Text>
             <InputTags.Tag.Close onClick={handleCloseTag} />
```

Comment:
`onClick={handleCloseTag}` why is it here? It doesn't do anything

Reply:
Same question

Reply:
Why? It deletes the tag when you click the cross. Clicking on tag sends tag to edit, but clicking on the cross deletes it